modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommerceTaxScreenNavigationConstants.java

```diff
@@ -30,6 +34,11 @@ public class CommerceTaxScreenNavigationConstants {
 	public static final String CATEGORY_KEY_COMMERCE_TAX_RATING_SETTINGS =
 		"tax-rate-settings";
 
+	public static final String ENTRY_KEY_COMMERCE_TAX_METHOD_AVALARA_RATE_RELS =
+		"avalara-tax-rates";
+
+	public static final String ENTRY_KEY_COMMERCE_TAX_METHOD_DETAIL = "detail";
+
```

Comment:
Hey @daiazius, Why did you put the ENTRY_KEY_COMMERCE_TAX_METHOD_DETAIL in this commit?